modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/cards.jsp

```diff
@@ -322,8 +322,9 @@
 		id="image-card-block"
 		md="6"
 	>
-		<clay:user-card-v2
+		<clay:user-card
 			actionDropdownItems="<%= cardsDisplayContext.getActionDropdownItems() %>"
+			labels="<%= cardsDisplayContext.getLabelItems() %>"
```

Comment:
Is this also possible through the `userCard` option?

I think we should add an example of `UserCard` usage which is in fact our most common pattern